apps/docs/components/docs/docs-feedback-widget.tsx

```diff
@@ -17,9 +17,8 @@ async function submitFeedback(
 	_feedback: 1 | -1 | 0,
 	_note: string
 ) {
-	const feedback: { value: number; message?: string } = { value: _feedback }
-	if (_note !== '') feedback.message = _note
-	track('Feedback', feedback)
+	// todo
+	await sleep(2000)
```

Comment:
?

Reply:
oh good catch, this was a bad merge, sorry!